chips/sam4l/src/spi.rs

```diff
@@ -464,6 +465,17 @@ impl SpiHw {
         // depending on the presence of the read/write below
         self.transfers_in_progress.set(0);
 
+        // Only setup the RX channel if we were passed a read_buffer inside
+        // of the option. `map()` checks this for us.
```

Comment:
Please add to this comment, explaining why this block of code needs to go before the other one. That way no one mistakenly moves it again when refactoring the code.

Reply:
I think this is addressed not, right?